paddle/scripts/docker/build.sh

```diff
@@ -185,14 +188,6 @@ EOF
     ${DOCKERFILE_GPU_ENV}
     ADD go/cmd/pserver/pserver /usr/bin/
     ADD go/cmd/master/master /usr/bin/
-EOF
```

Comment:
Cool, docs are pushed using travis-ci, so no need put in the `Dockerfile`.

And, do we need to put `make -j `nproc` print_operators_doc && paddle/pybind/print_operators_doc > doc/en/html/operators.json` in `paddle/scripts/travis`?